resources/js/Layouts/Authenticated.jsx

```diff
@@ -6,157 +6,166 @@ import ResponsiveNavLink from "@/Components/ResponsiveNavLink";
-                <NavLink
-                  href={route("dashboard")}
-                  active={route().current("dashboard")}
-                >
-                  Dashboard
-                </NavLink>
-              </div>
+							<div className="hidden space-x-8 sm:-my-px sm:ml-10 sm:flex">
+								<NavLink
+									href={route("dashboard")}
+									active={route().current("dashboard")}
+								>
+									Dashboard
+								</NavLink>
+							</div>
 
-              <div className="hidden space-x-8 sm:-my-px sm:ml-10 sm:flex">
-                <NavLink
-                  href={route("lessons")}
-                  active={route().current("lessons")}
-                >
-                  Lessons
-                </NavLink>
-              </div>
-            </div>
+							<div className="hidden space-x-8 sm:-my-px sm:ml-10 sm:flex">
+								<NavLink
+									href={route("categories")}
+									active={route().current("categories")}
+								>
+									Categories
+								</NavLink>
+							</div>
+						</div>
 
-            <div className="hidden sm:flex sm:items-center sm:ml-6">
-              <div className="ml-3 relative">
-                <Dropdown>
-                  <Dropdown.Trigger>
-                    <span className="inline-flex rounded-md">
-                      <button
-                        type="button"
-                        className="inline-flex items-center px-3 py-2 border border-transparent text-sm leading-4 font-medium rounded-md text-gray-500 bg-white hover:text-gray-700 focus:outline-none transition ease-in-out duration-150"
-                      >
-                        {`${auth.user.firstname} ${auth.user.lastname}`}
+						<div className="hidden sm:flex sm:items-center sm:ml-6">
+							<div className="ml-3 relative">
+								<Dropdown>
+									<Dropdown.Trigger>
+										<span className="inline-flex rounded-md">
+											<button
+												type="button"
+												className="inline-flex items-center px-3 py-2 border border-transparent text-sm leading-4 font-medium rounded-md text-gray-500 bg-white hover:text-gray-700 focus:outline-none transition ease-in-out duration-150"
+											>
+												{`${auth.user.firstname} ${auth.user.lastname}`}
 
-                        <svg
-                          className="ml-2 -mr-0.5 h-4 w-4"
-                          xmlns="http://www.w3.org/2000/svg"
-                          viewBox="0 0 20 20"
-                          fill="currentColor"
-                        >
-                          <path
-                            fillRule="evenodd"
-                            d="M5.293 7.293a1 1 0 011.414 0L10 10.586l3.293-3.293a1 1 0 111.414 1.414l-4 4a1 1 0 01-1.414 0l-4-4a1 1 0 010-1.414z"
-                            clipRule="evenodd"
-                          />
-                        </svg>
-                      </button>
-                    </span>
-                  </Dropdown.Trigger>
+												<svg
```

Comment:
separate a folder for your svg, make it a component so that you can customize properties

Reply:
Done creating separate components for svg files